src/geo_types_to_wkt.rs

```diff
@@ -400,7 +400,7 @@ where
 {
     let geo_types::MultiPoint(g_points) = g_mpoint;
     let w_points = g_points_to_w_points(g_points);
-    MultiPoint::from_points(w_points)
+    MultiPoint::from_points(w_points).unwrap_or(MultiPoint::empty(Dimension::XY))
```

Comment:
Oh yeah! I'm glad this is where we landed. I hadn't connected that this would panic for empty geo-types.